crates/pop-cli/src/commands/test/fast_forward.rs

```diff
@@ -312,6 +312,7 @@ mod tests {
 
 	#[tokio::test]
 	async fn fast_forward_snapshot_works() -> anyhow::Result<()> {
+		source_try_runtime_binary(&mut MockCli::new(), &crate::cache()?, true).await?;
```

Comment:
Fix. It was failing running this test isolated if try-runtimebinary was not in cache